menu.js

```diff
@@ -300,3 +309,38 @@ module.exports.setApplicationMenu = (win) => {
 	const menu = Menu.buildFromTemplate(menuTemplate);
 	Menu.setApplicationMenu(menu);
 };
+
+const iconPath = path.join(__dirname, "assets", "youtube-music-tray.png");
+const example = "Example: 'socks5://127.0.0.1:9999'";
+function setProxy(item, win) {
+	let options = {
+		title: 'Set Proxy',
+		label: 'Enter Proxy Address: (leave empty to disable)',
+		value: config.get("options.proxy") || example,
+		type: 'input',
+		inputAttrs: {
+			type: 'url'
+		},
+		icon: iconPath,
+		customStylesheet: "dark",
+		width: 450,
+	};
+	//TODO: custom bar on prompt need testing on macOS
+	if (!is.macOS()) {
```

Comment:
For the record: tested (without the `if`) on MacOS, no difference (maybe due to the custom titlebar plugin not working properly)

Reply:
Ok thats weird but kinda expected since you said in-app-menu doesn't really work on macOS